grammar.js

```diff
@@ -1227,7 +1243,19 @@ module.exports = grammar({
     ),
 
     c_struct_fields: $ => {
-      const names = seq($.identifier, repeat(seq(',', $.identifier)))
+      const names = seq(
```

Comment:
Outside of params, this and type declaration I don't expect to change. What are your thoughts on just adding these?

Reply:
I don't think I understand, which ones specifically?

Reply:
type declaration names and c struct / union fields